features/autoconsent.json

```
@@ -414,6 +414,10 @@
         {
             "domain": "ib3.org",
             "reason": "https://github.com/duckduckgo/privacy-configuration/pull/2702"
+        },
+        {
+            "domain": "bergwelten.com",
+            "reason": "Pay or Okay wall"
```

```suggestion
            "reason": "https://github.com/duckduckgo/privacy-configuration/pull/2706"
```

Could we use the PR link instead please?